### ISSUE TITLE
Add ADOPTERS.md

### ISSUE THREAD

**Issue by alexellis** [inlets/inlets#114]
Add Add ADOPTERS.md as per https://github.com/openfaas/faas/blob/master/ADOPTERS.md

**Comment by alexellis** [comment]
* Banzai cloud added
* VSHN added

https://github.com/inlets/inlets/blob/master/ADOPTERS.md

**Comment by ngxson** [comment]
Just a small (off-topic) question: I made a repo for deploying inlets to heroku (with click-to-deploy button for easier setup). Can it be added as a community tutorial? Thank you.
https://github.com/ngxson/inlets-heroku

**Comment by alexellis** [comment]
Sure, please do. Is this [based upon / inspired by my work with heroku](https://github.com/alexellis/inlets-heroku-tutorial)?

**Comment by ngxson** [comment]
No, I made from a template which support click-to-deploy to heroku. The inlets' binary also included in the repo, so it make setting up process very straightforward, specially for newbies.

**Comment by alexellis** [comment]
It would be better if you didn't commit the binary into your repo and have the user fetch it each time. 

You should find a section in the README for your link, feel free to send a PR and thank you for answering the questions.

**Comment by alexellis** [comment]
Do also see https://inlets.dev for other testimonials and adopters.
